remotes/transport/http.go

```diff
@@ -198,7 +204,11 @@ func (h *RemoteConnectionHandler) handleDeleteRemote(w http.ResponseWriter, r *h
 	}
 
 	if err := h.remotesService.DeleteRemoteConnection(r.Context(), *id); err != nil {
-		h.api.Err(w, r, err)
+		if strings.Contains(strings.ToLower(err.Error()), "foreign key constraint failed") {
```

Comment:
I spent a good amount of time trying to find an exact error to check with `errors.Is()` in `sqlite3`, but could not find one that matched. I think this should be done better but I'm not sure the best way to go about it

Reply:
Unless the API guarantees that the error types won't change, string matching is probably the better approach here. Worst case the string match fails and we get the 'new' underlying error.